### ISSUE TITLE
MPI and MUSIC 

### ISSUE THREAD

**Issue by drodarie** [dbbs-lab/bsb-nest#13]
If you install NEST with MUSIC and run bsb with MPI then you need to import nest before MPI. 
Unfortunately bsb imports MPI first.

**Comment by Helveg** [comment]
Yes and if you change the import order a whole bunch of other bugs appear. NEST and MUSIC should fix their MPI initialization logic ;) You can try it, but be prepared for other things to stop working, or subtle problems to appear, such as deadlocks.

**Comment by drodarie** [comment]
Yeah I do not want to dwell into this too but we might want to add a small comment on this in the documentation or at least keep this issue open?

**Comment by Helveg** [comment]
Moved to https://github.com/dbbs-lab/bsb/issues/35
